src/include/citus_config.h.in

```diff
@@ -31,14 +31,17 @@
 /* A string containing the version number, platform, and C compiler */
 #undef CITUS_VERSION_STR
 
+/* Define to 1 to build with lz4 support. (--with-lz4) */
+#undef HAVE_CITUS_LIBLZ4
+
 /* Define to 1 if you have the <inttypes.h> header file. */
 #undef HAVE_INTTYPES_H
 
 /* Define to 1 if you have the `curl' library (-lcurl). */
 #undef HAVE_LIBCURL
 
 /* Define to 1 if you have the `lz4' library (-llz4). */
-#undef HAVE_CITUS_LIBLZ4
+#undef HAVE_LIBLZ4
```

Comment:
These changes seem unrelated to your PR, lets not change the configure related files when we are not changing anything to the automake files.